stablehlo/transforms/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ add_mlir_dialect_library(StablehloPasses
   MLIRArithDialect
   MLIRAsmParser
   MLIRComplexDialect
+  MLIRDialectUtils
```

Comment:
We need to port this to BUILD.bazel too, can add around here

https://github.com/openxla/stablehlo/blob/6909631c594a0ac4f26a4e9bebbaa0f30740290c/BUILD.bazel#L964


```
 "@llvm-project//mlir:DialectUtils",
```